index.bs

```diff
@@ -1527,6 +1545,40 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Presentation Preference enumeration (enum <dfn enum>AttestationPresentationPreference</dfn>) ### {#attestation}
```

Comment:
there's various editorial things in the below that ought to be cleaned up but it's easier for me to do a PR than to try to write it all in comments -- so will endeavor to do such on the plane tomorrow.  see also: https://github.com/w3c/webauthn/pull/636#issuecomment-345196396